py/desispec/test/test_pipeline_graph.py

```diff
@@ -25,6 +25,7 @@ def setUp(self):
         self.raw = ph.fake_raw()
         self.redux = ph.fake_redux(self.prod)
         ph.fake_env(self.raw, self.redux, self.prod, self.redux)
+        self.specs = [ x for x in range(10) ]
```

Comment:
FYI, `list(range(10))` also works to convert the py3 iterator into a plain list.